pkg/core/bootstrap/autoconfig.go

```diff
@@ -3,36 +3,75 @@ package bootstrap
 import (
 	"fmt"
 	"io/ioutil"
-	"net/url"
 	"os"
-	"strings"
 
 	"github.com/pkg/errors"
 
 	"github.com/kumahq/kuma/pkg/config/api-server/catalog"
 
 	kuma_cp "github.com/kumahq/kuma/pkg/config/app/kuma-cp"
-	config_core "github.com/kumahq/kuma/pkg/config/core"
 	"github.com/kumahq/kuma/pkg/core"
 	"github.com/kumahq/kuma/pkg/tls"
 )
 
 var autoconfigureLog = core.Log.WithName("bootstrap").WithName("auto-configure")
 
 func autoconfigure(cfg *kuma_cp.Config) error {
+	if err := autoconfigureTLS(cfg); err != nil {
+		return err
+	}
+	autoconfigureServersTLS(cfg)
 	autoconfigureAdminServer(cfg)
 	autoconfigureCatalog(cfg)
 	autoconfigBootstrapXdsParams(cfg)
-	if err := autoconfigureKds(cfg); err != nil {
-		return err
+	return nil
+}
+
+func autoconfigureServersTLS(cfg *kuma_cp.Config) {
+	//if cfg.XdsServer.TlsCertFile == "" {
```

Comment:
Unneded?

Reply:
like todo said. Will be uncommented in the next PR which is coming today